app/views/layouts/_header.html.erb

```diff
@@ -154,7 +155,7 @@
       <% unless params[:action] == "register" || params[:action] == "signup" %>
         <ul class="nav navbar-nav navbar-right">
           <li class="dropdown">
-            <a onClick="$('#chat').toggle();$('#chat').append('<iframe width=\'400px\' height=\'500px\' style=\'border:0;\' src=\'https://chat.publiclab.org\'></iframe>');">
+            <a onClick="$('#chat').toggle();">
```

Comment:
Hi, @ananyo2012 !! great to see you. Here, we were trying not to actually instantiate the iframe until it's clicked. This change reverts that, so there could be some performance issues? (that was the original concern, anyways). Do you think we could set a local variable `true` on toggling, and only append if the variable is `!= true`?